new-frontiers-14/frontier-station-14
# Skipper Directional Fan Update

## About the PR



Don't blame me originally I just did the basic compliance stuff. A good soldier follows orders.

Replaced:
Tiny fans replaced with directional fans.
Plate vending machine replaced with service tech fab.
Single gas tank replaced with o2/n2 compliant mix.
Service EVA suit.

Added:
Limited material locker over service techfab.
Double fuel plasma locker.
Folding chair for cow convenience.
Wall mounted jukebox. 
Random wall poster to the dining area.
Techmaint under the airlock.

Changed:
Altered layout of the kitchen and bar area. Why do players need to jump onto a table to access a microwave. Why do people need to crawl over one another to find a chair.
Shifted the location of the chefdrobe to the freezer.
Shifted location of cow crate to botany to clear some space.
Updated power generation to default to 9k, closer to the ship's operating standard.
Removed and replaced blast doors and duplicate shutters over windows and the bar. Replaced with shutters.
Changed button locations.

## Why / Balance


Compliance. Demands from on high. 

## How to test


Check fans.
Check fuel locker.
Check mixer and atmos.
Don't fight the cow.
Check the generators to ensure their power output is good to go.

## Media

![ss+(2024-09-24+at+05 44 22)](https://github.com/user-attachments/assets/0394085b-4f5f-495e-a891-44f5cb13096d)


## Requirements


## Breaking changes


**Changelog**



:cl:
- tweak: Updated Skipper fans with directional version. Some changes to the internal configuration.
